cmd/dartboard/subcommands/load.go

```diff
@@ -28,6 +28,9 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+// TODO: Make this command idempotent. Get count (# of resources) matching some unique identifier.
+// Then rerun the appropriate script, passing in the index to leave off on.
+// * Scripts need to support this type of idempotency, they currently do not
```

Comment:
Fine to leave this, but let's also make this TODO an issue here in the repo so that it's easier to keep track of.

Reply:
https://github.com/rancher/dartboard/issues/106